plugins/ms-vscode.node-debug/1.0.0/meta.yaml

```diff
@@ -0,0 +1,13 @@
+id: ms-vscode.node-debug
+version: 1.0.0
```

Comment:
is the version accurate ?

Also how do we ensure that the version of ""vscode:extension/ms-vscode.node-debug" is always the same (and not being upgraded automatically).
For example it may work now but not after a new release of that vscode extension

Reply:
That's a good question. 
Initially I set the version of the extension, but Microsoft upgrades the extension automatically and after a while we will have inconsistency.
That's why I've decided to set `1.0.0` as a version.

Reply:
can we stick to a given version of microsoft extension ?

Reply:
Yes, that's possible If use direct url to the extension [1]

[1] https://marketplace.visualstudio.com/_apis/public/gallery/publishers/ms-vscode/vsextensions/node-debug2/1.31.6/vspackage